app/js/views/modal.jsx

```diff
@@ -21,3 +21,10 @@ export default class Modal extends React.Component {
     );
   }
 }
+
+Modal.propTypes = {
```

Comment:
Here are changes suggested by eslint react plugin, I actually like them. See [1] and [2].

[1] https://facebook.github.io/react/docs/reusable-components.html
[2] https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md